src/pages/removePlantPage/RemovePlant.tsx

```diff
@@ -13,16 +13,16 @@ import { toast } from "react-toastify";
 import Text from "../../components/text/Text.tsx";
 
 const RemovePlant = () => {
-  const { plantId } = useParams();
+  const { plantId } = useParams<{ plantId: string }>();
   const navigateTo = useNavigate();
   const queryClient = useQueryClient();
 
-  const removePlant = (id) => {
-    return pb.collection(PLANTS_COLLECTION).delete(id);
+  const removePlant = (id: string) => {
+    return pb.collection(PLANTS_COLLECTION).delete(id as string);
```

Comment:
hm why do you need type assertions here and below? You have just defined id as string. Also it seems you can just use the plantId from useParams?

Reply:
in line 20 if I remove type assertion, TS throws error that id has implicitly type any
in line 21, I removed it